ql/src/test/results/clientnegative/drop_invalid_constraint2.q.out

```diff
@@ -8,4 +8,4 @@ POSTHOOK: Output: database:default
 POSTHOOK: Output: default@table2
 PREHOOK: query: ALTER TABLE table1 DROP CONSTRAINT pk1
 PREHOOK: type: ALTERTABLE_DROPCONSTRAINT
-FAILED: Execution Error, return code 1 from org.apache.hadoop.hive.ql.ddl.DDLTask. InvalidObjectException(message:The constraint: pk1 does not exist for the associated table: default.table1)
```

Comment:
I think this is unintended. At least the test name suggests, that we want to test the constraints, not the tables